frontend/public/components/utils/kebab.tsx

```diff
@@ -106,7 +107,7 @@ const KebabItem_: React.FC<KebabItemProps & { isAllowed: boolean }> = ({
     </button>
   );
 };
-const KebabItemAccessReview_ = (props: KebabItemProps & { impersonate: string }) => {
+const KebabItemAccessReview_ = (props: KebabItemProps & { impersonate: ImpersonateKind }) => {
```

Comment:
This is weird, but tracking this change I think it's right. `string` was the wrong type.

Reply:
yeah, it was wrong ... :)